OilSandsConfig_en.ts

```diff
@@ -0,0 +1,608 @@
+const config = {
+    introSlide: {
+        logo: {
+            src: "" // TODO
+        },
+        title: "# NPRI Sector Overview: Oil Sands Extraction",
+        subtitle: "It also summarizes what steps facilities in this sector take to mitigate their environmental impacts"
+    },
+    slides: [
+        {
+            panel: [
+                {
+                    title: "## Overview",
+                    content:
+`**Oil Sands Extraction** is part of the National Pollutant Release Inventory's (NPRI) Sector Overview series. This sector overview explores NPRI substances released, disposed, and transferred by this industry. It also summarizes what steps facilities in this sector take to mitigate their environmental impacts.
+
+Businesses, institutions and other facilities across Canada must report their releases, transfers and disposals of pollutants to air, water and land annually to the Government of Canada's NPRI. The information collected is public, helps governments set environmental priorities and monitor environmental performance, and provides Canadians with an opportunity to learn about pollution in their surroundings.`,
+                    type: "text"
+                },
+                {
+                    src: "./images/NPRIpictogramme-2016data-EN__1553797637582__w1430.jpg",
+                    type: "image"
+                }
+            ]
+        },
+        {
+            panel: [
+                {
+                    title: "## Oil sands deposits",
+                    content:
+`There are three large oil sands deposits in Canada:
+
+- The Athabasca;
+- Peace River; and
+- Cold Lake.
+
+All three of the deposits are located in northern Alberta except for a small portion of the Cold Lake deposit, which extends into Saskatchewan. Together, these deposits cover [142,000 square kilometres](asdf) and are estimated to contain 166.3 billion barrels.
+
+The oil sands extraction sector is a major contributor to Canada’s economy, with more than 528,000 people deriving direct, indirect and induced employment from the oil sands and its supporting sectors. To date, [$243 billion of capital expenditures has been invested in the oil sands industry](https://www.nrcan.gc.ca/energy/energy-sources-distribution/crude-oil/what-are-oil-sands/18089), and the sector produces [2.9 million barrels of oil per day](https://www.nrcan.gc.ca/energy/energy-sources-distribution/crude-oil/oil-supply-and-demand/18086).
+
+This map shows the three oil sands areas. The actual geological deposits themselves cover larger areas.`,
+                    type: "text"
+                },
+                {
+                    config: {

```

Comment:
we probably want to figure out (maybe next meeting) if the map config should be included in the main config or if it should be in a separate folder and its own file (second approach might be cleaner for users just because of the length of configs already)

Reply:
yeah, i think this is worth a discussion. it might be easier to have it linked from its own file